irohad/network/consensus_gate.hpp

```diff
@@ -28,6 +28,21 @@ namespace shared_model {
 
 namespace iroha {
   namespace network {
+
+    enum class CommitType {
+      kVoted,  // peer voted for the same block
+      kOther   // peer has not voted for this block
```

Comment:
Don't like this names very much :( it is not clear without comments, what does `CommitType::kOther` means. Consider renaming, please

Reply:
For example, the enum class class itself can be renamed, such as `PeerVotedFor::{kThisBlock, kOtherBlock}`. This kind of radical solution, by, I think, more readable

Reply:
Good idea